whipper/common/encode.py

```diff
@@ -89,3 +90,61 @@ def _tag(self):
         w.save()
 
         self.stop()
+
+
+class EmbedPictureTask(task.Task):
+    description = 'Embed picture to FLAC'
+
+    def __init__(self, track_path, coverart_path):
+        self.track_path = track_path
+        self.coverart_path = coverart_path
+
+    def start(self, runner):
+        task.Task.start(self, runner)
+        self.schedule(0.0, self._embed_picture)
+
+    def _make_flac_picture(self, coverart_filename):
+        """Given a path to a jpeg or png file, return a flac.Picture
+
+        Returns: a valid flac.Picture or None
+        """
+        if not coverart_filename:
+            return None
```

Comment:
Nitpick: a simple `return` is fine.

Reply:
should i replace all the other `return None` too ?

Reply:
Are there other ones you've added? (in that case yes)